.github/workflows/linea-besu-package-release.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       release_tag_prefix: ${{ inputs.release_tag_prefix }}
       run_test: true
       run_e2e_test: ${{ github.event_name == 'workflow_dispatch' || needs.filter-commit-changes.outputs.versions_env == 'true' || needs.filter-commit-changes.outputs.docker-files == 'true' }}
-      push_image: ${{ github.event_name == 'workflow_dispatch' &&  inputs.skip_push_image != 'true' }}
+      push_image: ${{ (github.event_name == 'workflow_dispatch' || needs.filter-commit-changes.outputs.versions_env == 'true' || needs.filter-commit-changes.outputs.docker-files == 'true' || needs.filter-commit-changes.outputs.linea_configs == 'true') &&  inputs.skip_push_image != 'true' }}
```

Comment:
`versions_env` is sneak_case, when `docker-files` is kebab case. We should stick to 1 option. Because `versions-env`, which is also defined is not searchable as `versions_env`